### ISSUE TITLE
as a grant admin, i want to understand the funder funnel analytics, so i can see where there is drop off, and then use that information to reduce dropoff (and increase grant funding).

### ISSUE THREAD

**Issue by owocki** [gitcoinco/data-ops#36]
as a grant admin, i want to understand the funder funnel analytics, so i can see where there is drop off, and then use that information to reduce dropoff (and increase grant funding).

primer: i ( @owocki ) sent [this tweet](https://twitter.com/owocki/status/1085298325715206145) last night.  it performed well, got 30 likes, was RTd by Hudson Jameson, etc.

But despite all that there were NO conversions into new funders for the grant.

I want to know why.

Basically there are the following steps in funding a grant.
0. see a tweet or something about a gitcoin grant ( https://twitter.com/owocki/status/1085298325715206145 ) 
1. view grant
2. click 'fund grant'.
3. submit form
4. confirm tx's in metamask.

i would like to know the drop off is at each step, so we can optimize our conversion funnel.

**Comment by frankchen07** [comment]
Some initial reads:

1. https://insights.hotjar.com/sites/939263/funnel/responses/334475

Onboarding, looks like people don't tend to create grants off quickstart.

2. https://insights.hotjar.com/sites/939263/funnel/responses/334471

30% convert from `grant/new` to `grant/##/.*` (a newly created grant page). I'm not sure how to reconcile 72 sessions with only 17 grants made. I hypothesize that people may have been viewing the `grants/new` page, and then viewing already made grants, but that involves going to the explorer first, unless one has a direct link to the grant. That situation is not as likely, but I may be wrong.

With the assumptions above, we have a ~30% conversion rate (maybe a little less) from `grant/new` and have clicked create grant and confirmed tx's in metamask to get to `grant/##/.*` page. 

3. https://insights.hotjar.com/sites/939263/funnel/responses/334484 (Funding the grant from the grant details page).

4. https://insights.hotjar.com/sites/939263/funnel/responses/334485 (Funding the grant from the grant explorer).

Funnels 3 & 4 above aren't airtight because it's possible the user just clicked fund grant to see the form, and then hit the back button. The URL after successfully funding a grant doesn't change. Conversion looks to be in the ballpark of 4-8%.

```
https://gitcoin.co/grants/[0-9].*/.*
https://gitcoin.co/grants/[0-9].*/.*/fund
https://gitcoin.co/grants/[0-9].*/.* <- can be achieved by successful funding or the back button.
```

**Comment by owocki** [comment]
@frankchen07 sorry, should have been more clear... i was looking for a funnel analysis of *prospective funder* to *funder*, not grant creator (grant creators are *looking for funding*, not doing funding themselves)
